examples/cryptocurrency-advanced/README.md

```diff
@@ -58,35 +58,38 @@ Build the project:
 
 ```sh
 git clone https://github.com/exonum/exonum
-
 cd exonum/examples/cryptocurrency-advanced/backend
-
 cargo install --path .
 ```
 
-Generate template:
-
-<!-- markdownlint-disable MD013 -->
+Generate node configuration template:
 
 ```sh
 mkdir example
-
-exonum-cryptocurrency-advanced generate-template example/common.toml --validators-count 4
+exonum-cryptocurrency-advanced generate-template \
+  example/common.toml \
+  --validators-count 4
```

Comment:
```suggestion
  --validators-count 4 --supervisor-mode simple
```

Reply:
But there is a discussion on it [here](https://github.com/exonum/exonum/pull/1753#issuecomment-580293048).